assets/js/script.js

```diff
@@ -1,115 +1,98 @@
 var questions = [
```

Comment:
Consider moving less interesting information to another file or lower in the application - data (questions/answers) are unlikely to be more important than the code.